includes/page_top.php

```diff
@@ -3,6 +3,8 @@
     $page_request = $_GET['page_request'] ?? \FirstAide\Router::INDEX;
     $query = $_GET['query'] ?? '';
     $page = FirstAide\Router::getPage($UserObj, $page_request, $query);
+    
+    $page['javascripts'][] = 'error.js';
 
```

Comment:
`<title><?php` echo isset($_page['title']) ? $_page['title'].' | ' : ''  ?>FirstAide, Peacecorps</title>`
Let's add a default title in case $_page['title] is not set rather than keeping it empty.